### PR TITLE
feat(RELEASE-1203): create integration test ppl for release catalog

### PR DESCRIPTION
**Pull request by jinqi7**

### CODE REVIEW

File: integration-tests/pipelines/konflux-e2e-tests-pipeline.yaml
Comment (mmalina) [inline integration-tests/pipelines/konflux-e2e-tests-pipeline.yaml:1]: What's the purpose of this? Some description would be nice. Also, I find it a bit confusing that the top directory is integration-tests, but it's called e2e. So e2e or integration? Usually it's one or the other, no?

@flacatus maybe you can answer?

Comment (flacatus) [reply integration-tests/pipelines/konflux-e2e-tests-pipeline.yaml:1]: named integration-tests because is the name use konflux for tests pipelines

Comment (theflockers) [reply integration-tests/pipelines/konflux-e2e-tests-pipeline.yaml:1]: @flacatus can you please elaborate a bit more? I just read the ticket and I am also not getting what these tests will test - although yes, it is a integration one. Just to be more clear.

Comment (flacatus) [reply integration-tests/pipelines/konflux-e2e-tests-pipeline.yaml:1]: this Pull Request is just creating a pipeline to substitute openshift ci jobs. If you want you can rename the folder or put the pipeline somewhere else. Is up to you how you name it

Comment (flacatus) [reply integration-tests/pipelines/konflux-e2e-tests-pipeline.yaml:1]: maybe you can name it like: e2e/pipelines?

Comment (mmalina) [reply integration-tests/pipelines/konflux-e2e-tests-pipeline.yaml:1]: ok, thanks. now I get it - it's using the integration-test feature of Konflux, so hence the folder name. if that's the common place to have it, then it probably makes sense to use that.

can you please review the actual pipeline? I don't really know what should be in there.

Comment (johnbieren) [reply integration-tests/pipelines/konflux-e2e-tests-pipeline.yaml:1]: Yeah agree with Martin. It would be nice to have review from someone who has seen one of these files before, instead of us just blinding approving

**Review by flacatus — APPROVED** [review]

**Comment by flacatus** [comment]
My bad. Forgot to push approve button!

**Review by johnbieren — APPROVED** [review]
Marking approval per Flavius's approval

**Comment by flacatus** [comment]
@jinqi7 lets run an integration test from urbfork before merge

**Comment by jinqi7** [comment]
> @jinqi7 lets run an integration test from urbfork before merge

Sure. I am blocked by the [PR](https://github.com/konflux-ci/release-service-catalog/pull/613) which need to be merged first .

**Comment by johnbieren** [comment]
Commit needs to be signed

**Pull request merged**
